src/sentry/reprocessing2.py

```diff
@@ -351,8 +356,16 @@ def start_group_reprocessing(project_id, group_id, max_events=None, acting_user_
         data={"eventCount": event_count, "oldGroupId": group_id, "newGroupId": new_group.id},
     )
 
+    # New Activity Timestamp
+    date_created = new_activity.datetime
+
     client = _get_sync_redis_client()
     client.setex(_get_sync_counter_key(group_id), _REDIS_SYNC_TTL, event_count)
+    client.setex(
+        _get_info_reprocessed_key(group_id),
+        _REDIS_SYNC_TTL,
+        json.dumps({"dateCreated": date_created, "totalEvents": event_count}),
```

Comment:
Why are there two redis keys with the event_count in them?

Reply:
one of them is a countdown, so we decrease its value 😉